ibm/service/power/resource_ibm_pi_cloud_connection.go

```diff
@@ -224,10 +229,24 @@ func resourceIBMPICloudConnectionCreate(ctx context.Context, d *schema.ResourceD
 	}
 
 	client := st.NewIBMPICloudConnectionClient(ctx, sess, cloudInstanceID)
+	retryCount := 2
 	cloudConnection, cloudConnectionJob, err := client.Create(body)
```

Comment:
Better write a function and call Create() N no of times. Current logic tries 4 times overall.